File: frontend/src/app/model/notification.ts
export class Notification {
  notId : string;
  createdAt: Date;
  userId: string;
  message: string;
  status: boolean;

  constructor(
    notId : string,
    createdAt: Date,
    userId: string,
    message: string,
    status: boolean,

  ) {
    this.notId = notId;
    this.createdAt = createdAt;
    this.userId = userId;
    this.message = message;
    this.status = status;

  }
}

File: microservices/notifications-service/domain/notification.go
package domain

import (
	"github.com/gocql/gocql"
	"time"
)

type Notification struct {
	UserId         string     `json:"userId"`
	CreatedAt      time.Time  `json:"createdAt"`
	NotificationId gocql.UUID `json:"notificationId"`
	Message        string     `json:"message"`
	Status         string     `json:"status"`
}

type Notifications []*Notification
